pyrefly/lib/alt/class/class_field.rs

```diff
@@ -210,15 +210,26 @@ impl ClassField {
     }
 
     pub fn as_param(self, name: &Name, default: bool, kw_only: bool) -> Param {
-        let ClassField(ClassFieldInner::Simple { ty, .. }) = self;
+        let ClassField(ClassFieldInner::Simple { ty, annotation, .. }) = self;
+        
+        // For InitVar fields, use the inner type instead of InitVar[T]
+        let param_ty = if let Some(annotation) = &annotation
```

Comment:
I don't think you need the change to this file anymore; the if and else branches here are identical.